velox/exec/RowContainer.h

```diff
@@ -218,13 +220,31 @@ class RowColumn {
       return nullCount_ + nonNullCount_;
     }
 
+    void invalidateMinMaxColumnStats() {
+      minMaxStatsValid_ = false;
+    }
+
+    bool minMaxColumnStatsValid() const {
+      return minMaxStatsValid_;
+    }
+
+    void reset() {
+      minBytes_ = 0;
+      maxBytes_ = 0;
+      sumBytes_ = 0;
+      nonNullCount_ = 0;
+      nullCount_ = 0;
+      minMaxStatsValid_ = true;
+    }
+
     /// Merges multiple aggregated stats of the same column into a single one.
     static Stats merge(const std::vector<Stats>& statsList);
 
    private:
     // Aggregated stats for non-null rows of the column.
     int32_t minBytes_{0};
     int32_t maxBytes_{0};
+    bool minMaxStatsValid_{true};
```

Comment:
Do we want to just make minBytes_ and maxBytes_ wrapped around by std::optional?

Reply:
I tend to leave minMax name here for now to make it more explicit so people easily know what stats are invalidated. But in the future, if there are more stats that cannot be tracked, yes I think we should rename it.